src/UICrafter.Core/Components/BaseComponents/BaseLoadMetaDataList.razor

```diff
@@ -12,8 +12,9 @@ else
     }
     else
     {
+        <MudTextField @bind-Value="SearchString" Class="ma-2" Placeholder="Search" Adornment="Adornment.Start" Immediate="true" AdornmentIcon="@Icons.Material.Filled.Search" IconSize="Size.Medium"></MudTextField>
         <MudList T="AppView">
-            @foreach (var view in AppViews)
+            @foreach (var view in AppViews.Where(x => x.Name.ToLower().Contains(SearchString.ToLower())))
```

Comment:
You should do Contains(SearchString, InvariantCulture.IgnoreCase), or something along those lines.